#! /bin/sh /usr/share/dpatch/dpatch-run
## commit-email.dpatch by <peter@p12n.org>
##
## DP: Select the 'sendmail' interface for commit-email.pl by default,
## DP: which works better than the 'smtp_server' feature.

@DPATCH@
--- a/contrib/hook-scripts/commit-email.pl.in
+++ b/contrib/hook-scripts/commit-email.pl.in
@@ -57,8 +57,8 @@
 # leaving the other commented out, to select which method of sending
 # email should be used.
 # Using --stdout on the command line overrides both.
-#$sendmail = "/usr/sbin/sendmail";
-$smtp_server = "127.0.0.1";
+$sendmail = "/usr/sbin/sendmail";
+#$smtp_server = "127.0.0.1";
 
 # Svnlook path.
 my $svnlook = "@SVN_BINDIR@/svnlook";
